application/ImageSpinner.cpp

```diff
@@ -40,11 +40,11 @@ ImageSpinner::ImageSpinner(float const& imageHeight)
 	fBasicImageSpinner->SetExplicitMaxSize(BSize(B_SIZE_UNLIMITED, fImageHeight + 6));
 	fBasicImageSpinner->SetExplicitMinSize(BSize(0, 0/*fImageHeight + 6*/));
 
-	fDeleteButton = new ImageButton("quit", new BMessage(M_DELETE), 0.3, 1);
-	fBackButton = new ImageButton("back", new BMessage(M_MOVE_LEFT), 0.3, 1);
-    fNextButton = new ImageButton("next", new BMessage(M_MOVE_RIGHT), 0.3, 1);
+	fDeleteButton = new ImageButton("quit", new BMessage(M_DELETE), 0.3, 1, "Remove document");
+	fBackButton = new ImageButton("back", new BMessage(M_MOVE_LEFT), 0.3, 1, "Previous document");
+    fNextButton = new ImageButton("next", new BMessage(M_MOVE_RIGHT), 0.3, 1, "Next document");
```

Comment:
Alignment.

Reply:
They are aligned, don't know why github displays them like this.
I've also changed it to avoid line overflow.